upstairs/src/lib.rs

```diff
@@ -6080,12 +6085,12 @@ impl Buffer {
         Buffer::from_vec(vec)
     }
 
-    pub async fn len(&self) -> usize {
-        self.data.lock().await.len()
+    pub fn len(&self) -> usize {
+        self.len
     }
 
-    pub async fn is_empty(&self) -> bool {
-        self.len().await == 0
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
```

Comment:
Is this getting the length of `Buffer`?
Should this be. `self.len == 0` ?

Reply:
yeah, changed in a8d32c0